src/utils/serverCalls/getProducts.js

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+
```

Comment:
why getproduct method you have useaxios hook na

Reply:
There is naming issue. I using the getProducts function to  call it on CTA  you can't use hook inside click handlers and any conditional statement so thats the reason.